README.md

```diff
@@ -56,6 +56,7 @@ Clone or copy the source into your own project. There are sections marked with `
   * headers - added [helmet](https://www.npmjs.com/package/helmet)
   * CQRS - TODO
```

Comment:
remove

Reply:
done, this is removed as csrf covers this off